L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py

```diff
@@ -0,0 +1,228 @@
+#
```

Comment:
hi @davidlange6 
These files provide different modes of configuration of L1T Calo emulation. 
Apart from the default configuration, one can configure the emulator in any of the available modes via customisation functions.